src/types/http/THTTPExecuteParams.type.ts

```diff
@@ -13,9 +13,10 @@ limitations under the License.
 
 import * as http from "http";
 import * as https from "https";
+import { BodyInit } from "node-fetch";
 
 export type THTTPExecuteParams = {
-  body?: string;
+  body?: BodyInit | object;
```

Comment:
1. As far as I understand, BodyInit can have following types:
Blob | BufferSource | FormData | URLSearchParams | ReadableStream<Uint8Array> | string
And, object can be non-primitive type only.

It is definitely an improvement over very simplistic `string` type that we had earlier.
So, if I understand correctly, we still can't send Number etc.? - just wanted to confirm.
In Changelog, we have mentioned that 
```
and others as `application/octet-stream`
```
If my understanding is correct, Do we need to specify in docs what are the types NOT possible to be sent.

2. Instead of `BodyInit | object`, can we use `string | object` ?

Reply:
Good point, I included tests for Number and Boolean types now as well and they are passing